Add aria-label to My Account Navigation Item

masteradhoc opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:
Add the appropriate aria-label to the My Account Navigation Item
<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

Closes #43625 .

<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the My Account Page, Login
2. Check if the Aria-Label is now present

<!-- End testing instructions -->

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box and supplying data. -->

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

<!-- Choose only one -->

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->
Add aria-label to account page

#### Comment <!-- If the changes in this pull request don't warrant a changelog entry, you can alternatively supply a comment here. Note that comments are only accepted with a significance of "Patch" -->

</details>

masteradhoc committed:
add aria-label to My Account Navigation Item

masteradhoc commented:
@nielslange would it be possible to get this reviewed? :)

nielslange commented on plugins/woocommerce/templates/myaccount/navigation.php:
```diff
@@ -22,7 +22,7 @@
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account Pages">
```
```suggestion
<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```

Nitpick: The majority of `aria-label` in our codebase are using sentence case. What are your thoughts on using this here as well?

nielslange reviewed (changes requested):
Thanks for your contribution, @masteradhoc, and apologies for my delayed response. 

The change looks good. I just left a minor suggestion regarding using _"Account pages"_ instead of _"Account Pages"_.

While reviewing the PR, I noticed a few failing CI checks, which, I believe, you cannot see. 

1. **Highlight template changes**: As your change affects a template, the version number of the template should be bumped. This shows merchants/developers that the template changed. Currently, the version is set to `2.6.0`. It should be bumped to `9.0.0`, expecting that this change will be shipped with that release.
2. **Changelog entry**: It seems there was an issue with the changelog entry creation:
    - You've selected `Patch` and `Tweak - A minor adjustment to the codebase.` However, the `Patch` checkbox appears broken.
    - The option to `Automatically create a changelog entry from the details below` is unchecked. This checkbox triggers the workflow to automatically create the changelog entry.
    - Please add a simple changelog entry, such as "Add aria-label to account pages".

Could you please make these updates? Feel free to reach out if you have any questions or need further assistance!

masteradhoc committed:
Update plugins/woocommerce/templates/myaccount/navigation.php

Co-authored-by: Niels Lange <niels.lange@a8c.com>

masteradhoc committed:
bump version number for template

Commit:
Add changefile(s) from automation for the following project(s): woocommerce

masteradhoc commented:
> Thanks for your contribution, @masteradhoc, and apologies for my delayed response.
> 
> The change looks good. I just left a minor suggestion regarding using _"Account pages"_ instead of _"Account Pages"_.
> 
> While reviewing the PR, I noticed a few failing CI checks, which, I believe, you cannot see.
> 
> 1. **Highlight template changes**: As your change affects a template, the version number of the template should be bumped. This shows merchants/developers that the template changed. Currently, the version is set to `2.6.0`. It should be bumped to `9.0.0`, expecting that this change will be shipped with that release.
> 2. **Changelog entry**: It seems there was an issue with the changelog entry creation:
>    
>    * You've selected `Patch` and `Tweak - A minor adjustment to the codebase.` However, the `Patch` checkbox appears broken.
>    * The option to `Automatically create a changelog entry from the details below` is unchecked. This checkbox triggers the workflow to automatically create the changelog entry.
>    * Please add a simple changelog entry, such as "Add aria-label to account pages".
> 
> Could you please make these updates? Feel free to reach out if you have any questions or need further assistance!

@nielslange thanks for the help on this one! i went ahead and accepted your typo fix, adjusted the template version and fixed my changelog issues in the PR description. :)

nielslange commented:
Thank you so much for the quick changes, @masteradhoc. Due to an issue with the CI pipeline, I had to close and re-open the PR. So, please don't worry when you're seeing this. I'll wait until the CI pipeline is done and will do another review then.

barryhughes commented on plugins/woocommerce/templates/myaccount/navigation.php:
```diff
@@ -22,7 +22,7 @@
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```
Curious, should this be translated (as with other user-facing text)?

masteradhoc replied:
@barryhughes made the requested change - makes definately sense.

masteradhoc committed:
make string translateable

barryhughes reviewed (approved):
Great, thank you for making that adjustment :+1:

(No other notes from my end.)

nielslange commented:
@masteradhoc Thanks again for your contribution! 🙌 And thanks to @adimoldovan, who just merged it on my behalf.
